js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2425,7 +2425,7 @@ authorizationSignedResponseAlgHelp=JWA algorithm used for signing authorization
 associatedRolesRemoved=Associated roles have been removed
 keyAliasHelp=Alias for the private key
 whoWillAppearLinkTextUsers=Who will appear in this group list?
-tokenClaimName.tooltip=Name of the claim to insert into the token. This can be a fully qualified name like 'address.street'. In this case, a nested json object will be created. To prevent nesting and use dot literally, escape the dot with backslash (\\.).
+tokenClaimName.tooltip=Name of the claim to insert into the token. This can be a fully qualified name like 'address.street'. In this case, a nested JSON object will be created. To prevent nesting and use dot literally, escape the dot with backslash (\\.).
```

Comment:
```suggestion
tokenClaimName.tooltip=Name of the claim to insert into the token. This can be a fully qualified name such as 'address.street'. In this case, a nested JSON object is created. To prevent nesting and use dot literally, escape the dot with a backslash (\\.).
```